generator/generator.py

```diff
@@ -0,0 +1,34 @@
+#import numpy
+import sys
+import csv
+import random
+print("Hello")
+
+def generator(shards, deltaT, N, mean, std):
+    print("Hello")
+    tmp_List = []
```

Comment:
if you use "_" as word separator (we follow this style in python), don't use cammel case (java-like naming)
it should be tmp_list